app/src/main/res/layout/revision_card_fragment.xml

```diff
@@ -20,24 +20,24 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:orientation="vertical"
-      android:paddingBottom="160dp">
+      android:paddingBottom="@dimen/revision_card_fragment_padding_bottom">
 
       <TextView
         android:id="@+id/revision_card_explanation_text"
         style="@style/Body"
-        android:layout_marginStart="28dp"
-        android:layout_marginTop="36dp"
-        android:layout_marginEnd="28dp" />
+        android:layout_marginStart="@dimen/revision_card_fragment_layout_margin"
+        android:layout_marginTop="@dimen/revision_card_fragment_layout_margin_top_text"
+        android:layout_marginEnd="@dimen/revision_card_fragment_layout_margin" />
 
       <Button
         android:id="@+id/revision_card_return_button"
         style="@style/StateButtonActive"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_gravity="end"
-        android:layout_marginStart="28dp"
-        android:layout_marginTop="48dp"
-        android:layout_marginEnd="28dp"
+        android:layout_marginStart="@dimen/revision_card_fragment_layout_margin"
+        android:layout_marginTop="@dimen/revision_card_fragment_layout_margin_top_button"
```

Comment:
@ArchitJain1201 if instead of naming dimension as margin_top_button it will look more proper as buttom_margin_top, PTAL and change it along all the files, also in 1-2 files still the extra lines are not removed

Reply:
Sorry Ma'am, not able to get you, which extra lines you are talking about?